monarch/transitions/gui.lua

```diff
@@ -139,6 +145,9 @@ function M.create(node)
 			-- were transitioned out
 			if current_transition then
 				current_transition.fn(node, initial_data, current_transition.easing, 0, 0)
+				if current_transition.in_progress then
```

Comment:
So we call the transition function again on the line above to immediately finish it (duration = 0), but what if there already is a transition going? Won't the transition that was playing while the layout changed continue to play? Or does a call to gui.animate() override an existing animation? I think the default transitions need to do a gui.cancel_animation() as well or?

Reply:
Default transition will override by a new transition (with 0 duration) but without Done event because of nil callback of the new transition.